test/unit/validate.spec.js

```diff
@@ -10,10 +10,17 @@ describe('screener-storybook/src/validate', function() {
         });
     });
 
-    it('should throw error when no apiKey', function() {
-      return validate.storybookConfig({})
+    it('should throw error when no apiKey or username', function() {
```

Comment:
Can you add a test for the xor? That both apiKey and username/accessKey can't exist at the same time

Reply:
Added!